packages/auth-server/src/configuration/oidc/index.js

```diff
@@ -0,0 +1,71 @@
+import CONSTS from '@pins/common/src/constants.js';
+import { numberWithDefault } from '../config-helpers.js';
+import { dayInSeconds } from './ttl-defaults.js';
+import resourceFeature from './resources.js';
+import clientList from './clients.js';
+import Adapter from '../../adapter/prisma-adapter.js';
+import Account from '../../account/account.js';
+import logger from '../../lib/logger.js';
+
+let cookie_keys;
+try {
+	cookie_keys = JSON.parse(process.env.COOKIE_KEYS);
+	cookie_keys.forEach((x) => {
+		if (typeof x !== 'string') {
+			throw new Error('cookies.keys must be an array of strings');
+		}
+	});
+} catch (err) {
+	logger.warn({ err });
+	cookie_keys = [process.env.COOKIE_KEYS];
+}
+
+let jwks;
+try {
+	jwks = JSON.parse(process.env.JWKS);
+	if (!Array.isArray(jwks)) {
+		throw new Error('jwks must be a json array');
+	}
+} catch (err) {
+	logger.warn({ err }, 'invalid jwks array');
+	throw err;
+}
+
+const oidc = {
+	host: process.env.OIDC_HOST,
+	/** @type {import('oidc-provider').Configuration} */
+	configuration: {
+		clientAuthMethods: [CONSTS.AUTH.CLIENT_AUTH_METHOD],
+		clients: clientList,
+		claims: { allowed: ['email'] }, // not currently used, for opaque token + userinfo endpoint
+		cookies: {
+			keys: cookie_keys
+		},
+		jwks: {
+			keys: jwks
+		},
+		features: {
+			resourceIndicators: resourceFeature,
+			clientCredentials: {
+				enabled: true
+			},
+			devInteractions: {
+				enabled: false
+			}
+		},
+		ttl: {
```

Comment:
these time to live settings may need some thought, we can't have an access token that has session length, so users need to log in again once it expires.
Have gone with a day for now